sdk/authorization/mgmt/src/test/java/com/azure/resourcemanager/authorization/module/ApplicationsTests.java

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.resourcemanager.authorization;
+package com.azure.resourcemanager.authorization.module;
```

Comment:
The namespace conflict happens when we try to export test package. Same namespace not allowed here. For new plan to move testBase, it would not be required.